packages/screens/Rakki/components/BuyTickets/BuyTicketsButton.tsx

```diff
@@ -0,0 +1,62 @@
+import Long from "long";
+import { FC, useEffect, useState } from "react";
+import { View } from "react-native";
+
+import { BrandText } from "@/components/BrandText";
+import { Box } from "@/components/boxes/Box";
+import { CustomPressable } from "@/components/buttons/CustomPressable";
+import { Info } from "@/contracts-clients/rakki/Rakki.types";
+import { BuyTicketsModal } from "@/screens/Rakki/components/BuyTickets/BuyTicketsModal";
+import { neutral00, neutralFF, rakkiYellow } from "@/utils/style/colors";
+import { fontSemibold14 } from "@/utils/style/fonts";
+
+export const BuyTicketsButton: FC<{ networkId: string; info: Info }> = ({
+  networkId,
+  info,
+}) => {
+  const [isButtonHovered, setButtonHovered] = useState(false);
+  const [isModalVisible, setModalVisible] = useState(false);
+  const remainingTickets = info.config.max_tickets - info.current_tickets_count;
+  const [ticketAmount, setTicketAmount] = useState("1");
```

Comment:
Why this "amount" is in string format :o

Reply:
Sorry for adding these changes..
I didn't modify that is this PR, just splitted code from RakkiScreen.tsx.
This piece of code is the same as on origin/main: https://github.com/TERITORI/teritori-dapp/blob/main/packages/screens/Rakki/RakkiScreen.tsx#L119

Reply:
I forgot to remove this piece of code from here. See: https://github.com/TERITORI/teritori-dapp/pull/1434#discussion_r1898969767